airbyte-integrations/connectors/source-postgres/src/test-integration/java/io/airbyte/integrations/io/airbyte/integration_tests/sources/AbstractPostgresSourceDatatypeTest.java

```diff
@@ -726,29 +751,33 @@ private void addArraysTestData() {
             .addExpectedValues("[\"object\",\"integer\"]")
             .build());
 
-    addDataTypeTestData(
-        TestDataHolder.builder()
-            .sourceType("numeric_array")
-            .fullSourceDataType("NUMERIC[]")
-            .airbyteType(JsonSchemaType.builder(JsonSchemaPrimitive.ARRAY)
-                .withItems(JsonSchemaType.builder(JsonSchemaPrimitive.NUMBER)
-                    .build())
-                .build())
-            .addInsertValues("'{131070.23,231072.476596593}'")
-            .addExpectedValues("[131070.23,231072.476596593]")
-            .build());
+    for (final String type : Set.of("numeric", "decimal")) {
+      // This verifies NUMERIC/DECIMAL[] types to have
+      // default precision of (38, 0) if not specified
```

Comment:
whats the value thats being written in the posgtres source?

Reply:
CDC now preserves the decimal places if no Precision **_and_** Decimal are specified.